ov/profiles.go

```diff
@@ -505,6 +508,19 @@ func (c *OVClient) CreateProfileFromTemplate(name string, template ServerProfile
 	return err
 }
 
+func cleanup(template *ServerProfile) {
```

Comment:
Type should be ServerProfileTemplate.

Reply:
There is no server profile template struct in SDK

Reply:
Oh yes, we inherit all the attributes from ServerProfile itself in Golang. I forgot.

Reply:
@nabhajit-ray for Bios as well, in case you don't have Bios then pointer value will be nil, and, therefore, there might occur a panic.